source/command/analyzeCommand.ml

```diff
@@ -305,6 +312,16 @@ let with_performance_tracking ~debug ~f =
   result
 
 
+let verify_taint_configuration ~static_analysis_configuration () =
+  try TaintAnalysis.initialize_configuration ~static_analysis_configuration with
+  | Taint.TaintConfiguration.TaintConfigurationError _ as exn -> raise exn
+  | exn ->
+      (* The backtrace is lost if the exception is caught at the top level, because of `Lwt`. *
+         Let's print the exception here to ease debugging. *)
+      Log.log_exception "Taint analysis failed." exn (Worker.exception_backtrace exn);
```

Comment:
nitpick: Since we didn't do taint analysis here, maybe change the log message to something like "The taint configuration does not pass validation"?